classic_startup/python2.7libs/pythonrc.py

```diff
@@ -43,6 +44,26 @@ def classic_startup():
     # add to the system path
     sys.path.insert(0, tk_houdini_python_path)
 
+    # We need to check to make sure we don't have an incompatibility
+    # between httplib and Houdini's bundled ssl.py. This is a problem
+    # on some Linux distros (CentOS 7.x) with H16.
```

Comment:
Can you add a bit about how the setup of python path in houdini will bleed through to the system httplib which for older distros could be incompatible with the libraries from the houdini distribution?  The current comment does good for the what of what is happening, but there is some missing why (which down the road will be good to have when we want to remove this and don't remember why it is here).